docs/igor0806/GamesProject/games.html

```diff
@@ -0,0 +1,156 @@
+                                <a class="nav-link" href="./games.html">Games</a>
+                            </li>
+
+                            <li class="link">
+                                <a class="nav-link" href="./index.html">About</a>
+                            </li>
+
+                            <li class="link">
+                                <a class="nav-link" href="./form.html">Login</a>
+                            </li>
+
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </header>
+
+        <form class="form_games" action="#" data-type="filter-form">
+            <div class="container">
+                <div class="form_flex">
+                    <div class="select_gap">
+                        <div class="filter_select">
+                            <label for="Game_Genre"></label>
+                            <select id="Game_Genre" name="GenreSelect" class="select-name" data-type-filter="genre">
+                                <option value="Genre" disabled selected>Genre</option>
+                                <option value="Action">Action</option>
+                                <option value="Shooter">Shooter</option>
+                                <option value="ARPG">ARPG</option>
+                                <option value="Battle Royale">Battle Royale</option>
+                                <option value="Fighting">Fighting</option>
+                                <option value="Strategy">Strategy</option>
+                                <option value="MMORPG">MMORPG</option>
+                                <option value="Card Game">Card Game</option>
+                            </select>
+                        </div>
+
+                        <div class="platform-select">
+                            <label for="gamePlatform"></label>
+                            <select id="gamePlatform" name="PlatformSelect" class="select-name" data-type-filter="platform">
+                                <option value="Platform" disabled selected>Platform</option>
+                                <option value="PC">PC (Windows)</option>
+                                <option value="Web">Web Browser</option>
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="filter_radio">
+                        <input type="radio" class="radio" id="New" name="Radio" data-type-filter="new">
+                        <label for="New" class="New_radio">New</label>
+                        <input type="radio" id="Old" name="Radio" data-type-filter="old">
+                        <label for="Old" class="Old_radio">Old</label>
+                    </div>
+
+                    <div class="filter_search">
+                        <input type="text" id="search" name="search" placeholder="Search" class="search-input" data-type-filter="search">
```

Comment:
Why do you need an empty `label`? If it so necessary, you can wrap input by it
```suggestion
<label for="search">
  <input type="text" id="search" name="search" placeholder="Search" class="search-input" data-type-filter="search"></label>
```